olp-cpp-sdk-dataservice-read/src/repositories/CatalogCacheRepository.cpp

```diff
@@ -32,6 +34,8 @@
 // clang-format on
 
 namespace {
+constexpr char kCCRLogtag[] = "ApiCacheRepository";
```

Comment:
Shouldn't it be `CatalogCacheRepository`?
```suggestion
constexpr char kCCRLogtag[] = "CatalogCacheRepository";
```